showcase/src/assets/translations/nl.json

```diff
@@ -87,6 +87,17 @@
       "LOGOUT": {
         "LOGOUT_DEFAULT_ICON": "Logout-knop - Standaard pictogram",
         "LOGOUT_CUSTOM_ICON": "Logout-knop - Aangepast pictogram"
+      },
+      "COLLAPSIBLE": {
```

Comment:
I think it is good practice to order this json alphabetically :-) (even if we didn t do it 100% for now)